### PR TITLE
Show a loading bar while searching #172

### LINKED ISSUE #172: Show a loading bar while searching
I think it's important to add a loading bar so that the user knows whether the problem is from our side or from his side, because when the screen is all white, it is confusing whether there is a server fault or my browser isn't working.

* [ ] Design/ Choose the loading animation.
* [ ] Hook them To the section.
* [ ] Show it while loading the results

### PR DESCRIPTION
**Pull request by yasirunilan**
fixes #172

### CODE REVIEW

**Comment by jig08** [comment]
Demo link pls.

**Comment by yasirunilan** [comment]
wait I'll try to host it

**Comment by yasirunilan** [comment]
here is the link to the deployed app
https://yasirunilan.github.io/loklak_search/

Application search is not working.
In github deploying guide I tried step 5 
Step 5: As you search you might see that that it cant find anything, to resolve this, on search you will see there is a red shield on search bar, click on it and allow to load scripts

I Can't understand this step clearly

**Comment by Achint08** [comment]
@yasirunilan Which browser are you using?
If on firefox :- 
screen shot 2017-01-23 at 10 37 47 am
Open this info link and disable protection for now.

File: src/app/feed/feed.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 				[searchInputControl]="_queryControl"
 				(searchEvantEmitter)="handleSearchQuery()"></feed-header>
 
-	<div class="searching">
+	<div *ngIf="(isSearching$ | async)" class="searching" style="margin-top: 10%">
```
Comment (hemantjadon) [inline src/app/feed/feed.component.html]: No inline styles.

Comment (yasirunilan) [inline src/app/feed/feed.component.html]: I removed inline styles from the mentioned file. I'll make them committed to the same pull request. @hemantjadon what do you mean by unable to see the animation. What is shown to you when you hit enter after entering key word? Please clarify.

**Comment by yasirunilan** [comment]
I removed the inline styles. Please review

**Comment by yasirunilan** [comment]
@hemantjadon what do you mean by unable to see the animation. What is shown to you when you hit enter after entering key word? Please clarify.

File: src/app/feed/feed.component.scss [hunk 2]
```diff
@@ -53,3 +53,13 @@ app-footer.peeking{
...
+
+.searching {
+	border-radius: 50%;
+	margin: 10% auto;
+	width: 100px;
+	height: 100px;
+	border: 5px solid rgb(216,27,96);
+	border-top-color: rgba(216,97,96,0.2);
+	animation: spin 1s infinite linear;
+}
```
Comment (uttu357) [inline src/app/feed/feed.component.scss]: Also copy the spin animation from the feed-pagination scss file. That is why the animation is not working.

**Comment by uttu357** [comment]
Deploy the new changes https://yasirunilan.github.io/loklak_search/ so that it is easy to review and see them.

**Comment by Achint08** [comment]
Hello @yasirunilan . The PR was good. Animation was working fine now. Make a new PR if you wish to with the same code. and If you have faced any problem in deployment please share with us. We will guide you.

**Comment by yasirunilan** [comment]
@Achint08 I had a problem with deploying on github pages. SO I started with a fresh code checkout. You can review now. I have opened a new PR

**Pull request closed without merging**